scan-command/src/test/java/io/ballerina/scan/internal/ProjectAnalyzerTest.java

```diff
@@ -93,9 +85,21 @@ void testAnalyzingProjectWithCoreAnalyzer() {
 
     @Test(description = "Test analyzing project with external analyzers")
     void testAnalyzingProjectWithExternalAnalyzers() {
-        // TODO: Implement mock external analyzer issue assertions.
-        List<Issue> issues = projectAnalyzer.runExternalAnalyzers(project);
-        Assert.assertEquals(issues.size(), 0);
+        Assert.assertNotNull(projectAnalyzer);
```

Comment:
In which cases can this be `null ' except the RuntimeException occurred in the initialization? Isn't it itself a check? I mean do we really need this null check here as well?

Reply:
This was previously implemented to prevent spot bug errors. Tested after removing the null checks and it builds without issues.